graylog2-web-interface/src/views/components/aggregationwizard/__tests__/AggregationWizard.groupBy.test.tsx

```diff
@@ -31,6 +31,9 @@ import dataTable from 'views/components/datatable/bindings';
 
 import AggregationWizard from '../AggregationWizard';
 
+const timeout = (Number(process.env.TIMEOUT_MULTIPLIER) || 1) * 15000;
+jest.setTimeout(timeout);
```

Comment:
Instead of setting this globally, we can also pass the timeout as a third parameter to `it` (see [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts#L380)). That makes it a bit cleaner and decoupled from `jest`. I could also not say if calling `jest.setTimeout` in one test has an impact on timeouts in other tests.

Reply:
I like the approach, I will give it a try!

Regarding `jest.setTimeout`: "it only affects the test file from which this function is called"